src/Files.App/Views/ModernShellPage.xaml

```diff
@@ -51,12 +52,48 @@
 			Modifiers="Menu" />
 	</local:BaseShellPage.KeyboardAccelerators>
 
-	<Frame
-		x:Name="ItemDisplayFrame"
-		HorizontalAlignment="Stretch"
-		x:FieldModifier="public"
-		BorderBrush="{x:Bind CurrentInstanceBorderBrush, Mode=OneWay}"
-		BorderThickness="{x:Bind CurrentInstanceBorderThickness, Mode=OneWay}"
-		Navigated="ItemDisplayFrame_Navigated" />
+	<Grid HorizontalAlignment="Stretch" wct:UIElementExtensions.ClipToBounds="True">
+		<Grid
```

Comment:
Does ellipse support child elements? If it does, maybe we can eliminate this grid.

Reply:
Ellipse isn't a ContentControl so it doesn't, but the ellipse can be eliminated by setting the same properties on the grid instead.

Reply:
What about `Border`?

Reply:
I've changed it to Border